source/nvwave.py

```diff
@@ -907,20 +907,31 @@ def feed(
 		@param size: The size of the data in bytes if data is a ctypes pointer.
 			If data is a Python bytes object, size should be None.
 		@param onDone: Function to call when this chunk has finished playing.
-		@raise WindowsError: If there was an error playing the audio.
+		@raise WindowsError: If there was an error initially opening the device.

```

Comment:
does this still get raised elsewhere? just to confirm

Reply:
as-in, after these changes, is this still raised in this function?

Reply:
1. feed() calls open(), which can raise WindowsError.
2. Previously, even if open() succeeded, feed() could also itself raise WindowsError.
3. 2) is no longer possible.
4. However, feed() can still raise WindowsError indirectly because of 1).
5. This new behaviour is consistent with WinmmWavePlayer.